server.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+var express = require('express');
+var app = express();
+var server = require('http').createServer(app);
+var Grid = require('./src/grid.js');
+var Cell = require('./src/cell.js');
+
+app.use(express.static(__dirname + '/public'));
+
+var grid;
+
+app.get('/', function (req, res) {
+  grid = new Grid(Cell)
+  grid.init();
+  res.sendFile(__dirname + '/views/index.html');
+});
+
+app.post('/', function (req, res) {
```

Comment:
Although this is a short server file, it's always a good idea to abstract this into a routes file. Keep this file as minimal as possible